nexus/src/app/background/init.rs

```diff
@@ -1023,6 +1027,9 @@ pub struct BackgroundTasksData {
     pub saga_recovery: saga_recovery::SagaRecoveryHelpers<Arc<Nexus>>,
     /// Channel for TUF repository artifacts to be replicated out to sleds
     pub tuf_artifact_replication_rx: mpsc::Receiver<ArtifactsWithPlan>,
+    /// Channel for exposing the latest loaded blueprint
+    pub blueprint_load_tx:
```

Comment:
Just making sure I understand: we have this field now because in `nexus/app/mod.rs`, we need to construct this channel to get a handle to the receiver before we've set up the background task system?